scripts/build-utils.mjs

```diff
@@ -43,10 +43,10 @@ export function exec(command, args = []) {
   }
 
   return new Promise((resolve, reject) => {
-    const child = spawn(command, args);
-
-    child.stdout.pipe(process.stdout);
```

Comment:
Is the child process standard and error output lost now?

Reply:
No. Instead of piping, the stdout and stderr manually and ignoring stdin, the child process now inherits stdio (stdin, stdout and stderr) from the parent process.

The output is not lost and we can actually now see colors in the child process output when using `exec(...)` from `build-utils`.
![image](https://github.com/web-eid/web-eid-webextension/assets/5690464/bbb7007f-4ed7-4901-8c12-e4f619fb2bb2)